tests/manage/storageclass/test_create_sc_and_make_it_as_a_default.py

```diff
@@ -0,0 +1,119 @@
+import pytest
+import logging
+from ocs_ci.ocs import ocp
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs.resources.pod import get_fio_rw_iops
+from tests.fixtures import create_project
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.usefixtures(
+    create_project.__name__,
+)
+class TestCreateStorageClassandMakeItAsDefault(ManageTest):
+    """
+    Verifies that a storageclass can be made as default
+    storageclass and pvc based on this storageclass
+    can be successfully used to create an app pod and
+    run IOs from it.
+    """
+    @pytest.mark.parametrize(
+        argnames="interface_type",
+        argvalues=[
+            pytest.param(
+                *[
+                    constants.CEPHBLOCKPOOL
+                ], marks=pytest.mark.polarion_id("OCS-626")
+            ),
+            pytest.param(
+                *[
+                    constants.CEPHFILESYSTEM
+                ], marks=pytest.mark.polarion_id("OCS-627")
+            )
+        ]
+    )
+    def test_create_sc_and_make_it_as_a_default(
+        self, interface_type, storageclass_factory,
+        pvc_factory, pod_factory
+    ):
+        """
+        Test function which verifies the above class
+        """
+        # Get default StorageClass
+        initial_default_sc = ocp.get_default_storage_class()
+        log.info(f"Current default StorageClass is:{initial_default_sc}")
+
+        # Create a Storage Class
+        sc_obj = storageclass_factory(interface=interface_type)
+        log.info(
+            f"{interface_type}StorageClass: {sc_obj.name} "
+            f"created successfully"
+        )
+
+        # Change the above created StorageClass to default
+        log.info(
+            f"Changing the default StorageClass from {initial_default_sc}"
+            f" to {sc_obj.name}"
+        )
+        ocp.change_default_storageclass(scname=sc_obj.name)
+        # Confirm that the default StorageClass is changed
+        tmp_default_sc = ocp.get_default_storage_class()
+        log.info(f"Current Default StorageClass is:{tmp_default_sc}")
+        assert tmp_default_sc == sc_obj.name, (
+            "Failed to change default StorageClass"
+        )
+        log.info(
+            f"Successfully changed the default StorageClass to "
+            f"{sc_obj.name}"
+        )
+
+        # Create a PVC using the default StorageClass
```

Comment:
from this point on, it makes this test tier2

Reply:
Done